python_modules/libraries/dagster-polars/dagster_polars/io_managers/delta.py

```diff
@@ -412,11 +412,11 @@ def get_metadata(
                 # but it's unclear how to get the length of the partition we are appending to
                 pass
             else:
-                metadata["append_row_count"] = metadata["row_count"]
+                metadata["append_row_count"] = metadata["dagster/row_count"]
```

Comment:
should there be a defensive check that this key is in `metadata` ? is it certain to always be present?

Reply:
This key should always be there, yes